wurf_runner.py

```diff
@@ -244,9 +244,19 @@ def assemble_output(stdout, stderr):
     """Helper function to assemble output message from the test results"""
     msg = []
     if stdout:
+        try:
+            stdout.encode("ascii")
+        except UnicodeError:
+            stdout = stdout.encode("utf-8")
         msg.append('\nstdout:\n\n{}'.format(stdout))
```

Comment:
Why don't we go for utf-8 by default?
It would be interesting to test the output with a Unicode string, maybe on different platforms.